kuzzleio/kuzzle-vuejs-typescript-boilerplate
# Add backend instance folder as a submodule

## What does this PR do ?
Adds the backend instance folder as a submodule from the **kuzzle-plugin-advanced-boilerplate**

### How should this be manually tested?

`docker-compose -f backend/docker-compose.yml up`
`cd frontend`
`npm install && npm run serve`

And check everything is working like a charm!

### Boyscout
* Add github issue and pull request  template
* Fix linter error in frontend
* Fix command to run kuzzle in readme
